lte/gateway/Vagrantfile

```diff
@@ -75,7 +75,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     magma_trfserver.vm.provider "virtualbox" do |vb|
       vb.name = "magma-trfserver"
       vb.linked_clone = true
-      vb.customize ["modifyvm", :id, "--memory", "256"]
+      vb.customize ["modifyvm", :id, "--memory", "1024"]
```

Comment:
why is that needed?

Reply:
Ubuntu does need more memory. The machine did not even start with 256.